requirements/requirements-base.txt

```diff
@@ -1 +1,2 @@
+Django==1.10.6
```

Comment:
Sure - how should the version of django be specified in the requirements then? Otherwise, when working locally you need to install requirements **and** django.